doc/en/customize.rst

```diff
@@ -97,6 +97,19 @@ check for ini-files as follows::
 .. _`how to change command line options defaults`:
 .. _`adding default options`:
 
+Change which files are added to sys.path
+-----------------------------------------------
+
+If you place a conftest.py file in the root directory of your project
+(as determined by pytest, see above.) pytest will run tests against 
+the code below that directory by adding it to your sys.path instead of 
```

Comment:
After a quick look I couldn't find any evidence in the sources of that... perhaps @RonnyPfannschmidt can chime in?